server/lobbyconnection.py

```diff
@@ -971,3 +968,30 @@ def send(self, message):
         if self.player:
             self._logger.debug("Lost lobby connection removing player {}".format(self.player.id))
             self.player_service.remove_player(self.player)
+
+    async def abort_connection_if_banned(self):
+        async with self._db.acquire() as conn:
+            now = datetime.now()
+            result = await conn.execute(
+                select([ban.c.reason, ban.c.expires_at]).where(
+                    ban.c.player_id == self.player.id and ban.c.expires_at < now
```

Comment:
Here is mistake, use `and_`
Use filter instead of where

Reply:
Good catch